src/Xhgui/Searcher/Mongo.php

```diff
@@ -304,10 +273,12 @@ protected function _wrap($data)
         }
 
         if (is_array($data)) {
+            $data['_id'] = (string) $data['_id'];
```

Comment:
This cast could be done in Profile as well. Either the constructoror on get.

Reply:
Right, done https://github.com/perftools/xhgui/pull/244/commits/975dbddee3a6872bee48f650fbb6f6d64fa37144